### ISSUE TITLE
Do not wrap exceptions

### ISSUE THREAD

**Issue by qoomon** [rainerhahnekamp/sneakythrow#1]
Hi,

thx for your library however I would appreciate if you wouldn't wrap exceptions.
You can throw checked exceptions with following usage of generics.

```java
public class ExceptionUtils {
    
    /**
     * Throws {@code t} as a unchecked exception.
     */
    @SuppressWarnings("unchecked")
    public static <T extends Throwable> void throwUnchecked(Throwable t) throws T {
        throw (T) t;
    }


    public static void main(String[] args) {

      Exception e = new Exception("Boom!");

      throwUnchecked(e);
    }
    
}
```

**Comment by rainerhahnekamp** [comment]
@qoomon can you check if it is like you've suggested?

**Comment by qoomon** [comment]
yes, looks good to me.
However i would call the method throwUnchecked instead of rethrow 😄

**Comment by rainerhahnekamp** [comment]
Fixed in #2
